### DISCUSSION TITLE
Rename "windows" to "panes" (or "tiles")

### DISCUSSION THREAD

**Discussion by roadkell** [kovidgoyal/kitty#3551] [Ideas]
It might be better to call window frames inside kitty "panes" or "tiles", for better consistency across applications (terminal emulators, mutiplexers, and other software), and to avoid confusion with OS windows concept.

Also, thanks for the great job!

**Comment by BMJ-pdx** [comment]
I heartily agree (I prefer "panes").  But unfortunately the term "window" is almost as established as Benjamin Franklin's determination of the direction of electrical current flow :-)
I suggest the rigorous inclusion of "OS" in the term "OS window".
Of course, if someone's in favor of %s/window/pane/g , I'm all for it.

**Comment by egnor** [comment]
\+ :100: to this, as a newb who was confused by this a bunch. I think "pane" would be a great name.

In addition to using "OS window" to refer to OS windows, I'd suggest that documentation should liberally say things like "window (Kitty pane)" or the like, not necessarily on every usage but somewhere within view.

**Reply by kovidgoyal** [reply]
I dont think i want to add a new term for something for which a perfectly good term already exists. However, I am sympathetic to the potential for confusion, so you are welcome to send PRs for the docs (in the docs folder) changing references to the word "window" to something like :term:`kitty window<window>`. Which will be rendered as a link with the text "kitty window" which when clicked on will take you to the glossary definition of the term.

**Reply by kovidgoyal** [reply]
And I made an initial commit to get you started, creating the glossary. https://github.com/kovidgoyal/kitty/commit/9e9d1a4e96bdddb93b4096d85e66abda06ba8efa
